### DISCUSSION TITLE
Sectors with attitude "faults" occasionally appear. How to deal with them?

### DISCUSSION THREAD

**Discussion by ChaiWenyu1983** [filecoin-project/lotus#9225] [General Q&A] [answered]
Sectors with attitude "faults" occasionally appear and continue to increase. Some can be recovered automatically the next day, while others cannot. What is the way to handle them manually?

**Comment by rjan90** [comment, marked as answer]
Hey @ChaiWenyu1983! As you already may know the sectors are marked as faulty if you are unable to prove the sectors during windowPoSt. They will automatically turn back to proving if you are able to prove them in next windowPoSt for that window (24 hours later). It is not possible to manually get them back to the proving state.

I would try to search your logs for why your sectors are being marked as faulty though, as there might be some bottlenecks or issues in your setup.
